### DISCUSSION TITLE
Copied to clipboard button with swap

### DISCUSSION THREAD

**Discussion by alibugrat** [saadeghi/daisyui#751] [Q&A]
Hi,

 I want to make copied to clipboard button with swap. I am not sure that it's best way to make this. Is there any chance to turn back earlier swap after a few seconds? 
Is there any chances to control swap in the script?

**Comment by saadeghi** [comment]
You can add/remove the class name or check/uncheck the checkbox using JS after a few seconds.

This is not possible with pure CSS of course.
